bin/default_pythons

```diff
@@ -2,7 +2,7 @@
 
 DEFAULT_PYTHON_VERSION="python-3.6.8"
 
-if [[ $PLOTLY_IS_AIRGAPPED -eq 1 ]]; then
+if [[ "$PLOTLY_IS_AIRGAPPED" == "1" ]]; "then
```

Comment:
Please remove the extra quote before the `then` :)

Reply:
:see_no_evil: cp/paste typo. Thanks!